drivers/ethernet/Kconfig.enc424j600

```diff
@@ -0,0 +1,45 @@
+# Kconfig - ENC424J600 Ethernet driver configuration options
+
+#
+# Copyright (c) 2019 PHYTEC Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig ETH_ENC424J600
+	bool "ENC424J600C Ethernet Controller"
+	depends on SPI
+	help
+	  ENC424J600C Stand-Alone Ethernet Controller
+	  with SPI Interface
+
+config ETH_ENC424J600_RX_THREAD_STACK_SIZE
+	int "Stack size for internal incoming packet handler"
+	depends on ETH_ENC424J600
```

Comment:
instead of the "depends on", you could use if ETH_ENC424J600 around the group of options.

Reply:
fixed